### PR TITLE
Issue #724: [cocoa] Shell.open() does not always cause Shell to activate

### PR DESCRIPTION
**Pull request by SyntevoAlex**
fixes https://github.com/eclipse-platform/eclipse.platform.swt/issues/724

### CODE REVIEW

**Comment by SyntevoAlex** [comment]
In master now.

File: tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/SwtTestUtil.java [hunk 1]
```diff
@@ -396,26 +396,29 @@ public static boolean waitEvent(Runnable trigger, Control control, int swtEvent,
...
-	final int timeout = 3000;
-	final long timeBeg = System.currentTimeMillis();
+	final int timeout = 1000;
+
+	Runnable trigger2 = () -> {
+		if (isCocoa) {
+			// Issue #731: if another app gains focus during entire JUnit session,
+			// newly opened Shells do not activate. The workaround is to activate
+			// explicitly.
+			shell.forceActive();
```
Comment (HeikoKlare) [inline tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/SwtTestUtil.java:406]: Taking a look at this change, I do not really understand why it should change anything.
In most cases, the `waitShellActivate` method is called for shells that are closed. But calling `forceActive` on a closed shell has no effect. Do I miss something?

Comment (SyntevoAlex) [reply tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/SwtTestUtil.java:406]: You are right, that was a mistake. It should go after original trigger. Thanks for the catch!

Comment (HeikoKlare) [reply tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/SwtTestUtil.java:406]: I give it a try in #741.

Comment (SyntevoAlex) [reply tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/SwtTestUtil.java:406]: Thanks!

**Pull request merged**